podcasts/podcasts.go

```diff
@@ -520,7 +520,7 @@ func (p *Podcasts) PurgeOldPodcasts(maxAge time.Duration) error {
 	expDate := time.Now().Add(-maxAge)
 	var episodes []*db.PodcastEpisode
 	err := p.db.
-		Debug().
+		Where("status = ?", "completed").
```

Comment:
does this also work?

```suggestion
		Where("status = ?", db.PodcastEpisodeStatusCompleted).
```